src/clap.rs

```diff
@@ -316,6 +316,11 @@ use CPU (3) as the default instead.
         .help("Sets the default widget type, use --help for info.")
         .long_help(DEFAULT_WIDGET_TYPE_STR);
 
+    let expanded_on_startup = Arg::new("expanded_on_startup")
+        .long("expanded")
+        .short('e')
+        .help("Expanded selected widget upon starting the app. Same as pressing \"e\" inside the app. Use with \"default_widget_type\" and \"default_widget_count\" to select desired expanded widget.");
```

Comment:
nit: would move this to `long_help`, I usually prefer to try to keep `help` shorter and concise.